chef/data_bags/crowbar/bc-template-cinder.json

```diff
@@ -19,6 +19,9 @@
       "use_gitbarclamp": true,
       "use_virtualenv": true,
       "use_pip_cache": true,
+      "min_pool_size": 10,
+      "max_pool_size": 100,
+      "pool_timeout": 2,
```

Comment:
Two questions:
 - are we sure these defaults are good defaults?
 - shouldn't we make these settings available in all barclamps? (they're just in neutron right now)